### PR TITLE
Feature/backend/api/import vcard

### PR DESCRIPTION
**Pull request by lisonferez**
New route in API Rest: POST /import.
This route make it possible to import a file of vcards and insert new contact at a user with vcards.

### CODE REVIEW

**Comment by gdchamal** [comment]
I think a rebase from master is missing or your branch doesn't start from master but your previous one (for import vcard documentation) as we see commit not related to this API feature

**Comment by gdchamal** [comment]
Also the modified swagger.json file containing your new route is missing in this PR

**Comment by gdchamal** [comment]
:+1:

File: src/backend/defs/rest-api/paths/import.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  post:
+    consumes:
+    - text/vcard
+    - text/vcf
+    - text/x-vcard
+    responses:
+      '200':
+        description: Import successful
+      '400':
+        description: Syntax error
```
Comment (sapiens-sapide) [inline src/backend/defs/rest-api/paths/import.yaml]: Miss : 
 * "security" directive
 * "produces" directive
 * "description" & "schema" for responses.

I suggest the route to be `/v1/contacts/import` or  `/v1/import/contacts`

Comment (gdchamal) [inline src/backend/defs/rest-api/paths/import.yaml]: Different content-type permit to import differents format.
I think a generic route is not so bad

**Pull request merged**